pcsx2/gui/GlobalCommands.cpp

```diff
@@ -70,10 +70,13 @@ namespace Implementations
 		g_Conf->EmuOptions.GS.FrameSkipEnable = !g_Conf->EmuOptions.GS.FrameSkipEnable;
 		SetGSConfig().FrameSkipEnable = g_Conf->EmuOptions.GS.FrameSkipEnable;
 
-		if( EmuConfig.GS.FrameSkipEnable )
+		if( EmuConfig.GS.FrameSkipEnable ) {
+			OSDlog( wxColor(L"RED"), "(FrameSkipping) Enabled : FrameDraws=%d, FrameSkips=%d", g_Conf->EmuOptions.GS.FramesToDraw, g_Conf->EmuOptions.GS.FramesToSkip );
```

Comment:
I don't like this kind of copy/past in the log. Maybe it would be nicer to extend OSDlog to also print on the Console.